### DISCUSSION TITLE
Is it a bug that all the important dependencies are being installed by create-next-app into the "devDependencies"?

### DISCUSSION THREAD

**Discussion by mike-rambil** [vercel/next.js#56983] [Help] [answered]
### Summary

Below are the files that are being installed in my devDependencies Array.

```
"devDependencies": {
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "autoprefixer": "^10",
    "eslint": "^8",
    "eslint-config-next": "13.5.4",
    "postcss": "^8",
    "tailwindcss": "^3",
    "typescript": "^5"
  }
```

### 
**Is This Normal ???**

**Comment by icyJoseph** [comment, marked as answer]
There has been a bit of left, right, swings on this matter:

Basically, since the output mode could do it by just looking at dependencies, it was done that way:

- https://github.com/vercel/next.js/pull/48076#pullrequestreview-1376312311
- https://github.com/vercel/next.js/issues/43066#issuecomment-1319969015

However, it'd seem that there was more discussions within the Next.js team:

- https://github.com/vercel/next.js/pull/55730

And as a result of that, we now have it back to `dependencies` and `devDependencies`.

**Reply by mike-rambil** [reply]
Thanks! Good to know :)
